src/model/database/db_build.sql

```diff
@@ -0,0 +1,49 @@
+BEGIN;
+
+DROP TABLE IF EXISTS users,articles,contacts,chats CASCADE;
+
+CREATE TABLE users (
+  user_id SERIAL PRIMARY KEY ,
+	user_name VARCHAR(100) NOT NULL,
+	user_email TEXT NOT NULL UNIQUE,
+	user_password TEXT,
+	permission  VARCHAR(100) DEFAULT 'user'
```

Comment:
definitely you won't have a role with 100 character also the same for the username